docs/how-tos/event-logging.md

```diff
@@ -18,21 +18,25 @@ Eppo's SDKs include either an assignment logger base class or an interface, in w
+| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`               |
+| `featureFlag` (string)    | An Eppo feature flag key                                                                                                 | "recommendation-algo"               |
+| `allocation` (string)     | An Eppo allocation key                                                                                                   | "allocation-17"                     |
 
-Eppo expects that the logger function will take this object and write data back to your warehouse in a format that roughly matched the table below (the specific column names do not matter).
+Eppo expects that the logger function will take this object and write data back to your warehouse in a format that roughly matched the table below. The specific column names do not matter, but the columns are needed to [define assignments](/data-management/definitions/assignment-sql.md).
 
-| experiment            | subject | variation | timestamp                  | subject_attributes    |
-| :-------------------- | :------ | :-------- | :------------------------- | :-------------------- |
-| `recommendation_algo` | `6141`  | `control` | `2021-06-22T17:35:12.000Z` | `{ "country": "US" }` |
+| experiment                          | subject | variation | timestamp                  | subject_attributes    |
+| :---------------------------------- | :------ | :-------- | :------------------------- | :-------------------- |
+| `recommendation-algo-allocation-17` | `6141`  | `control` | `2021-06-22T17:35:12.000Z` | `{ "country": "US" }` |
 
-It's ok for this table to contain duplicate rows for the same subject. Further, if a subject is assigned to mulitple variants, Eppo will automatically remove them from the analysis.
+It's ok for this table to contain duplicate rows for the same subject. If a subject is assigned to mulitple variants, Eppo will automatically remove them from the analysis.
+
+The following fields are optional to write to your warehouse, not required by Eppo, and provided mainly for convenience: `featureFlag` and `allocation`. When you call a function to get the assignment, the `experiment` key is generated by combining the `featureFlag` and `allocation` fields. If `allocation` is the key of an experiment allocation, then the assignment data within this experiment key is valid for experiment analysis by Eppo.
```

Comment:
Is this easy to understand/follow?

Reply:
I think so (then again we've been knee deep in this for a while 😅)